dsl/src/main/scala/com.crobox.clickhouse/dsl/TableColumn.scala

```diff
@@ -5,14 +5,14 @@ import com.crobox.clickhouse.dsl.marshalling.QueryValue
 import com.crobox.clickhouse.dsl.schemabuilder.{ColumnType, DefaultValue}
 import com.dongxiguo.fastring.Fastring.Implicits._
 
-sealed case class EmptyColumn() extends TableColumn("")
+case object EmptyColumn extends TableColumn("NULL")
```

Comment:
Should we rename this to null column in this case?

Reply:
We could; Ideally we get rid of this thing; it feels a bit like Option, but then misused in certain places

Reply:
Reason i made it NULL is that empty string is not a valid name for a column; since i added the require(isValidIdentifier) since someone showed me that using ref() you can add `(` and `)` to GROUP BY queries so it already supports `GROUP BY ROLLUP(a,b,c)`